docs/index.md

```diff
@@ -62,7 +62,7 @@ Go try it **now** online in a Jupyter notebook: [![Open In Colab](https://colab.
 * [Ελληνικά](Tutorial-gr.md)
 * [עברית](Tutorial-he.md)
 * [简体中文](Tutorial-zh.md)
-* [বাংলা](Tutorial-%E0%A6%AC%E0%A6%BE%E0%A6%82%E0%A6%B2%E0%A6%BE.md)
```

Comment:
Thanks!

There is also another broken link on line 59, a little above this one 😅
`Tutorial-हिंदी.md` should be renamed. Could you please fix it too?

Reply:
:P .. Missed it again. Thank You. Changes pushed.